src/sinks/util/service.rs

```diff
@@ -166,7 +168,7 @@ const fn default_retry_attempts() -> Option<usize> {
 }
 
 const fn default_retry_max_duration_secs() -> Option<u64> {
-    Some(3_600)
+    Some(60)
```

Comment:
We could reduce even more e.g. 30 seconds.

Reply:
Yeah I'll change it. I'm in favor of being more aggressive here.